src/main/java/com/example/event_manager/service/EventServiceImpl.java

```diff
@@ -49,4 +51,18 @@ public Event findById(final Long id) {
 
     return tasks;
   }
+
+  @Override
+  public Map<String, List<Event>> getEventsPartition() { // It uses repo to call spring data jpa methods (3 queries) not yet paginated
```

Comment:
[Not an issue]
You were curios in different way of doing that.
You can also use CASE WHEN to calculate 1 column for 3 values -> started, notstarted, outdated then group it to map.
https://www.postgresql.org/docs/9.5/functions-conditional.html

Reply:
Yea I did not know about CASE WHEN